src/frontend/components/LocalizedTimedTextTrackUpload/LanguageSelect/index.tsx

```diff
@@ -5,7 +5,7 @@ import { defineMessages, useIntl } from 'react-intl';
 import { useJwt } from 'data/stores/useJwt';
 import { useTimedTextTrackLanguageChoices } from 'data/stores/useTimedTextTrackLanguageChoices';
 import { useTimedTextTrack } from 'data/stores/useTimedTextTrack';
-import { LanguageChoice } from 'types/LanguageChoice';
+import { LanguageChoice } from 'types/SelectOptions';
 import { timedTextMode } from 'types/tracks';
 
 const messages = defineMessages({
```

Comment:
line 65, you have to sort the `availableSelectableLanguages` based on the language label. The `Select` component will not sort alphabetically the language list:

```
return (
      choices &&
      choices.filter((lang) => !excludedLanguages.includes(lang.value))
        .sort((a, b) => {
          return a.label.localeCompare(b.label);
        })
    );
```